records_mover/records/delimited/hints.py

```diff
@@ -19,34 +22,82 @@ class Hints:
     timeonlyformat = LiteralHint[HintTimeOnlyFormat](HintTimeOnlyFormat,  # type: ignore
                                                      'timeonlyformat',
-                                                     default="HH24:MI:SS")
+                                                     default="HH24:MI:SS",
+                                                     description=("Format used to write "
+                                                                  "'time' values"))
+    # https://docs.python.org/3/library/csv.html#csv.Dialect.doublequote
     doublequote = LiteralHint[HintDoublequote](HintDoublequote,  # type: ignore
                                                'doublequote',
-                                               default=False)
+                                               default=False,
+                                               description=('Controls how instances of quotechar '
+                                                            'appearing inside a field should '
+                                                            'themselves be quoted. When True, the '
+                                                            'character is doubled. When False, the '
+                                                            'escapechar is used as a prefix to the '
+                                                            'quotechar.'))
     header_row = LiteralHint[HintHeaderRow](HintHeaderRow,  # type: ignore
                                             'header-row',
-                                            default=True)
-    quotechar = StringHint('quotechar', default='"')
-    record_terminator = StringHint('record-terminator', default='\n')
-    field_delimiter = StringHint('field-delimiter', default=',')
+                                            default=True,
+                                            description=('True if a header row is provided in '
+                                                         'the delimited files.'))
+    # https://docs.python.org/3/library/csv.html#csv.Dialect.quotechar
+    quotechar = StringHint('quotechar',
+                           default='"',
+                           description=('A one-character string used to quote fields containing '
+                                        'special characters, such as the delimiter or quotechar, '
+                                        'or which contain new-line characters.'))
+    record_terminator = StringHint('record-terminator',
+                                   default='\n',
+                                   description='String used to close out individual rows of data.')
+    field_delimiter = StringHint('field-delimiter',
+                                 default=',',
+                                 description='Character used between fields.')
+
+
+# TODO: Shoudl this be a method on DelimitedRecordsFormat as well?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker